core/src/main/scala/kafka/coordinator/group/GroupCoordinator.scala

```diff
@@ -613,16 +624,29 @@ class GroupCoordinator(val brokerId: Int,
         // The group is only using Kafka to store offsets.
         // Also, for transactional offset commits we don't need to validate group membership and the generation.
         groupManager.storeOffsets(group, memberId, offsetMetadata, responseCallback, producerId, producerEpoch)
-      } else if (group.is(CompletingRebalance)) {
```

Comment:
Could we get some new test cases since we have changed the order here?

Reply:
Sure